tools/genkconfig.py

```diff
@@ -101,6 +101,7 @@ def main():
         "servers/apps": "Applications",
         "servers/drivers/blk": "Block Drivers",
         "servers/drivers/net": "Network Drivers",
+        "servers/drivers/time": "Timer Drivers",
```

Comment:
Nit: `Timer Driver` -> `Real Time Clock Driers`

A *timer* sounds similar to RTC, but they're quite different.

Reply:
Makes sense. Changed.